src/components/uploadFile/uploadFile.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './uploadFile.scss';
+
+//material-components
+import Button from '@material-ui/core/Button';
+import CloudUpload from '@material-ui/icons/CloudUpload';
+
+class UploadFile extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = { file: null };
+        this.onFileChange = this.onFileChange.bind(this);
```

Comment:
this binders are not longer required, you can write your functions as:
`onFileChange = (e) => { ... }`